arch/arm/src/common/arm_initialize.c

```diff
@@ -34,7 +34,12 @@
 
 /* g_interrupt_context store irq status */
 
-#if defined(CONFIG_ARCH_ARM)
```

Comment:
why not keep CONFIG_ARCH_ARM

Reply:
Done, ARCH_ARM not old arm arch, let me remove this check